kubernetes-dbless/docker/node.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/sh
```

Comment:
pq a t-on besoin de ce script ? J'ai du mal à voir les différences, et idéalement on n'aurait pas deux scripts à maintenir.
(même remarque pour le docker file : il faudrait construire au dessus de l'autre)

Reply:
c'était juste pour tester et demontrer notre mode standalone avec Kube. 
En soi, ça devrait même pas être merger mais c'était pour voir si tout fonctionnait

Reply:
- node.sh:
  - Correctif d'un bug dans le script (manquait un -n sur la commande New-Node)
  - Desactivation de l'authentication du WS (le `enableWsApiAuth=false` de la config ne suffisait pas)
- Dockerfile:
  - Pointe vers la config standalone 
  - Changement du nombre de pools max
  - Suppression du healthcheck

En soit modulo la config utilisee qui ajoute `wsStandlaneMode=true` on a juste besoin de la commande du node.sh qui desactive l'authentication.

Reply:
Commit https://github.com/enioka-Haute-Couture/jqm/pull/541/commits/78740ddbb5742e6e8ac6b05680e2dbf1163fef34 properly integrates the standalone deployment into the existing docker stuff